src/math/p5.Vector.js

```diff
@@ -1654,62 +1882,71 @@ p5.Vector = class {
   }
 
   /**
- * Linear interpolate the vector to another vector.
+ * Calculates new `x`, `y`, and `z` components that are proportionally the
+ * same distance between two vectors. The `amt` parameter is the amount to
+ * interpolate between the old vector and the new vector. 0.0 keeps all
+ * components equal to the old vector's, 0.5 is halfway between, and 1.0 sets
+ * all components equal to the new vector's.
```

Comment:
Do you think it'd be helpful to mention p5's `lerp()` for non vectors? Maybe how this is kind of like calling that on each axis? Or is that too much

Reply:
The documentation is very similar, so I'm not sure if this would clarify or add context.